p4-16/psa/PSA.mdk

```diff
@@ -896,8 +897,9 @@ packet contents and metadata when a packet begins egress processing.
 +--------------+-------------+---------------+--------------+-----+
 | `packet_path`    | Same value as received by EgressParser above. ||||
 +--------------+-------------+---------------+--------------+-----+
-| `instance`         | From PacketReplicationEngine configuration for NM packets. ||||
-|                    | 0 for all other kinds of packets.                          ||||
+| `instance`         | 0 | from PRE           | 0 for cloned packets that are not multicast replicated. ||
+|                    |   | configuration      | from PRE configuration of multicast group  ||
+|                    |   | of multicast group | for cloned packets that are multicast replicated ||
```

Comment:
I'm not sure I follow the text in the CI2E column. Maybe prepend a 'value from PRE configuration ...'